recipe/meta.yaml

```diff
@@ -17,22 +17,20 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy <1.14  # [py27]
```

Comment:
:thinking: can you do `numpy != 1.14.0` lol?!

Reply:
Good question. Should I try?

Reply:
:woman_shrugging:

Reply:
I did something but i have no idea what

Reply:
it appears to have worked!

Reply:
circle passed.

Reply:
doesnt that just mean the recipe has no issues with it?

Reply:
it means it built on linux.

Reply:
Shrug